index.bs

```diff
@@ -1898,58 +1775,54 @@ unless the user agent also implements the URL [[URL]] specification.
 In other words, <code>URL.prototype</code> must evaluate to true if the user agent implements the URL [[URL]] specification,
```

Comment:
This "ECMAScript user agents" section seems unnecessary these days, where everyone has a `URL` interface.